### PR TITLE
CLDC-1389: Set default option on accessible autocomplete

### PR DESCRIPTION
**Pull request by baarkerlounger**
image

### CODE REVIEW

**Review by dushan-madetech — APPROVED** [review]

**Comment by paulrobertlloyd** [comment]
Erm… I think the default, when enhanced, should be an empty value.

**Comment by baarkerlounger** [comment]
@paulrobertlloyd cool, could you comment that on the ticket then please?

https://digital.dclg.gov.uk/jira/browse/CLDC-1379

File: app/frontend/controllers/accessible_autocomplete_controller.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ export default class extends Controller {
     const rawFieldName = matches ? `${matches[1]}[${matches[2]}_raw]` : ''
 
     accessibleAutocomplete.enhanceSelectElement({
-      defaultValue: '',
+      defaultValue: options[0].name,
```
Comment (smeny) [inline app/frontend/controllers/accessible_autocomplete_controller.js]: Is it guaranteed that options[0] is always present?

**Comment by paulrobertlloyd** [comment]
> @paulrobertlloyd cool, could you comment that on the ticket then please?

Updated the ACs.

**Comment by baarkerlounger** [comment]
@smeny @dushan-madetech @paulrobertlloyd 

Changed this to filter out the "Select an option" value (so it doesn't appear if you start typing "Se..." either, and left the box blank by default when JS is enabled.

**Pull request merged**